sway/commands/clipboard.c

```diff
@@ -0,0 +1,38 @@
+#include <wlc/wlc.h>
+#include <unistd.h>
+#include <string.h>
+#include "sway/commands.h"
+#include "stringop.h"
+
+static void send_clipboard(void *data, const char *type, int fd) {
+	if ((strcmp(type, "text/plain") != 0) &&
+			(strcmp(type, "text/plain;charset=utf-8") != 0)) {
+		close(fd);
+		return;
+	}
+
+	const char *str = data;
+	write(fd, str, strlen(str));
```

Comment:
It could work both ways and I'd err on the side of no null delimiter, but so long as you've tested it this way then I trust that it works.